src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        for (int i = 0; i < 3; i++) {
+            Lottery lottery = new Lottery();
+            Ball randomBall = lottery.getRandomBall();
+            System.out.println("randomBall_" + i + " " + randomBall.toString());
```

Comment:
```suggestion
            System.out.println("randomBall_" + i + " " + randomBall);
```

we don't need to call toString method explicitly it will be called "under the hood"